src/application/monitoring/monitoring.use-case.ts

```diff
@@ -23,58 +34,76 @@ export class MonitoringUseCase {
    * - 알림 생성(TODO)
    * 을 차례로 수행하는 유즈케이스
    */
-  async recordLiveStreams() {
-    // 구독 스트리머 목록이 변경되었는지 여부 확인
-    let liveStreamers: LiveStreamerResult[] = [];
-    if (this.isChanged) {
-      console.log('구독 현황 변경!');
-      this.isChanged = false;
+  async recordLiveStreams(): Promise<void> {
+    try {
+      await this.transactionManager.beginTransaction(async () => {
```

Comment:
@smileboy0014 
Prisma $transaction API의 [Interactive transactions](https://www.prisma.io/docs/orm/prisma-client/queries/transactions#interactive-transactions)은 아래와 같이 콜백의 인자로 `PrismaClient`인스턴스를 받아서 사용합니다.

```ts
$transaction<R>(fn: (prisma: PrismaClient) => R): R
```
따라서 아래와 같이 콜백 인자로 트랜잭션 인스턴스를 전달받도록 수정이 필요해 보입니다🙂
```suggestion
    try {
      await this.transactionManager.beginTransaction(async (tx) => {
```

Reply:
아 넵 저도 이 문제를 나중에 발견하였습니다.. prisma에서 트랜잭션의 단위를 묶으려면 따로 트랜잭션을 만들어서, 그 트랜잭션 내에서 함수를 호출해야 한다고 하더라구요.... 이 방식으로 적용하려면 모든 메서드마다 tx를 인자로 넘겨줘야 해서 그 방식을 적용하려면 공수가 너무 많이 드는것 같다는 생각이 들었습니다.
그래서 이 pr 이후 한번 더 이슈를 만들어서 관련 헬퍼 함수를 쉽게 쓰기 위해 리팩토링 하고자 합니다!
- #35

Reply:
좋습니다 🙏🏻